rviz_default_plugins/src/rviz_default_plugins/robot/robot_link.cpp

```diff
@@ -179,6 +179,8 @@ void RobotLinkSelectionHandler::postRenderPass(uint32_t pass)
   }
 }
 
+static std::map<const RobotLink *, std::string> errors;
```

Comment:
I have to say that this seems like a weird way to do this.  Can't we just have `std::string error` as a member variable in the `RobotLink` class?

Reply:
https://github.com/ros2/rviz/pull/744/commits/a9a84609447f36f7800758bed2c0d39be0e531c7